package.json

```diff
@@ -2,11 +2,16 @@
   "name": "redux-promise-actions",
   "version": "0.5.1",
   "description": "Flux Standard Action async utilities for Redux.",
-  "main": "index.js",
+  "main": "dist",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "babel src --presets env --out-dir dist",
     "lint": "eslint src/**/*.js",
+    "prepublish": "npm run build",
```

Comment:
Can you change this to `prepublishOnly` please? Otherwise it runs with `npm install`, because npm :\

You can read more about that change here: https://docs.npmjs.com/misc/scripts#prepublish-and-prepare